presto-native-execution/README.md

```diff
@@ -223,7 +221,7 @@ headers and clang-tidy warnings.  These targets are shortcuts for calling
 
 GitHub Actions run `make format-check`, `make header-check` and
 `make tidy-check` as part of our continuous integration.  Pull requests should
-pass linux-build, format-check, header-check and other jobs without errors
+pass Linux-build, format-check, header-check and other jobs without errors
```

Comment:
```suggestion
pass linux-build, format-check, header-check and other jobs without errors
```
I think in the context of the job name that this should be `linux-build`. I looked at https://github.com/prestodb/presto/blob/6e87912f1e80ba202fad2f4ea7d0ab1072c9d515/.circleci/continue_config.yml#L26. Let me know if you're referring to another job.